### DISCUSSION TITLE
Is there a way to make the screenshot scrollable?

### DISCUSSION THREAD

**Discussion by f00dog-x300** [pytest-dev/pytest-html#551] [Q&A]
I'm writing an appium test so it uses a mobile screen. However, when I take a screenshot of the app, it only shows part of the screen. I need to be able to scroll down and view the screen or double-click the screen to view it. Is there a way to do that?

**Comment by BeyondEvil** [comment]
What happens if you click the image?

**Reply by f00dog-x300** [reply]
Nothing happens. However, if I right-click and select "Open image in new tab" in Chrome then it shows the full image

**Reply by BeyondEvil** [reply]
Ok, I'll take this to next-gen and see if we can implement that clicking an image opens it in a new tab.

Would that be an expected behaviour?

**Reply by f00dog-x300** [reply]
I would think so yeah. It looks find if I can take a screenshot from a browser, but when I use it on my mobile framework, it only shows the top part of the screen. For reference this is the code I use to pick up the screenshot in the `conftest.py` 
```
@pytest.hookimpl(hookwrapper=True, tryfirst=True)
def pytest_runtest_makereport(item: pytest.Item, call: pytest.CallInfo) -> None:  # pylint: disable=unused-argument
    """Sets the result of each test in the report."""
    pytest_html = item.config.pluginmanager.getplugin("html")
    outcome = yield
    report_path = "reports"

    report = outcome.get_result()
    # # set an report attribute for each phase of a call
    setattr(item, "rep_" + report.when, report)
    extra = getattr(report, "extra", [])

    if report.when == "call":
        feature_request = item.funcargs["request"]
        driver = feature_request.getfixturevalue("driver")
        nodeid = item.nodeid
        xfail = hasattr(report, "wasxfail")

        if (report.skipped and xfail) or (report.failed and not xfail):
            file_name = f"{nodeid}_{datetime.today().strftime('%Y-%m-%d_%H_%M')}.png".replace(
                "/", "_").replace("::", "_").replace(".py", "")
            img_path = os.path.join(report_path, "screenshots", file_name)
            driver.save_screenshot(img_path)
            screenshot = driver.get_screenshot_as_base64()
            extra.append(pytest_html.extras.image(screenshot, ""))
        report.extra = extra
```

**Comment by BeyondEvil** [comment]
Can you checkout 4.0.0rc0 and see if the behaviour is what you expect?

**Comment by BeyondEvil** [comment]
4.0.0rc1 is out now
